Packs/DeveloperTools/Integrations/XsoarServerPowershell/XsoarServerPowershell.yml

```
@@ -0,0 +1,39 @@
+category: Messaging
+commonfields:
+  id: XsoarServerPowershell
+  version: -1
+configuration: []
+description: This integration allows interacting with XSOAR server.
+display: XSOAR Server Powershell
```

@GalRabin let's change the display name:
```suggestion
display: XSOAR Powershell Testing
```

---

You will need to also update the display name in the release notes and detailed description.

---

Changed in all required places.